PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,108 @@
+<!--
+  PLEASE READ THE FIRST SECTION :-)
+-->
+
+### Is this a bug report?
+
+(write your answer here)
+
+<!--
+  If you answered "Yes":
+  
+    Please note that your issue will be fixed much faster if you are
+    including the exact reproduction steps and a demo.
+    
+    If you're in a hurry or don't feel confident, it's fine to report bugs with
+    less details, but this makes it less likely they'll get fixed soon.
+
+    In either case, please fill as many fields below as you can.
+
+  If you answered "No":
+
```

Comment:
Here, I would rather have it urge the user to join the Slack Channel (https://react-chat-signup.herokuapp.com) or post to Stack overflow with the `react-static` tag.